apps/innovations/_services/innovation-supports.service.ts

```diff
@@ -933,13 +917,15 @@ export class InnovationSupportsService extends BaseService {
     domainContext: DomainContextType,
     support: string | InnovationSupportEntity,
     accessorRoleIds: string[],
+    message: string,
+    changedStatus: boolean,
     threadId?: string,
     entityManager?: EntityManager
   ): Promise<{ newAssignedAccessors: string[]; removedAssignedAccessors: string[] }> {
     // Force a transaction if one not present
     if (!entityManager) {
       return this.sqlConnection.transaction(async transaction => {
-        return this.assignAccessors(domainContext, support, accessorRoleIds, threadId, transaction);
+        return this.assignAccessors(domainContext, support, accessorRoleIds, message, false, threadId, transaction);
```

Comment:
Shouldn't you pass `changedStatus` instead of `false`?

Reply:
Ops! Glad you saw this one!
Thanks!